python/ray/experimental/client/worker.py

```diff
@@ -53,9 +54,13 @@ def __init__(self,
         else:
             self.channel = grpc.insecure_channel(conn_str)
         self.server = ray_client_pb2_grpc.RayletDriverStub(self.channel)
+
         self.data_client = DataClient(self.channel, self._client_id)
         self.reference_count: Dict[bytes, int] = defaultdict(int)
 
+        self.log_client = LogstreamClient(self.channel)
+        self.log_client.set_logstream_level(logging.INFO)
```

Comment:
Can we also call setup_logger() from ray_logging to get a consistent log format on the client and server?